app/concerns/koudoku/subscription.rb

```diff
@@ -35,7 +35,7 @@ def processing!
 
             # update the package level with stripe.
             if sub = customer.subscriptions.first
-              sub.prorate = Koudoku.prorate
+              sub.prorate = Koudoku.prorate || (upgrading? ? Koudoku.prorate_for_upgrade : Koudoku.prorate_for_downgrade)
```

Comment:
Why is that?  
For me it's more like, if the prorate option is defined to true then it prorate all the time. Otherwise it uses the prorate_for_upgrade/downgrade thingy. If you do it the opposite way if prorate_for_upgrade or prorate_for_downgrade it false, Koudoku.prorate will be used. That's not what we want.